### PR TITLE
Update TaskData and TaskController

### PR DESCRIPTION
**Pull request by Tzirliya**
Add updates based on comments from previous pull request. A new constructor with Entity as a parameter is added to TaskData. A method toEntity() is added to TaskData. Quantity and sort is added to getTasks() in TaskController. A filter for specific keys is added to the query in deleteTasks() in TaskController.

### CODE REVIEW

File: src/main/java/com/rtb/projectmanagementtool/task/TaskController.java [hunk 1]
```diff
@@ -67,14 +56,13 @@ public void addTasks(HashSet<TaskData> tasks) {
   public void deleteTasks(HashSet<Long> taskIDs) {
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     HashSet<Key> keys = new HashSet<>();
-    Query query = new Query("Comment").setKeysOnly();
+    Query query = new Query("Task");
+    Filter filter = new Query.FilterPredicate("__key__", Query.FilterOperator.IN, taskIDs);
+    query.setFilter(filter);
```
Comment (jcespejo) [inline src/main/java/com/rtb/projectmanagementtool/task/TaskController.java:61]: I'm curious if we can take a list of taskIDs and build keys using the [KeyFactory](https://cloud.google.com/appengine/docs/standard/java/javadoc/com/google/appengine/api/datastore/KeyFactory) so we wouldn't have to query for the keys first before deleting.

File: src/main/java/com/rtb/projectmanagementtool/task/TaskController.java [hunk 2]
```diff
@@ -17,23 +22,22 @@ public TaskController(HashSet<TaskData> tasks) {
...
-      String name = (String) entity.getProperty("name");
-      String description = (String) entity.getProperty("description");
-      Status status = (Status) entity.getProperty("status");
-      HashSet<Long> users = (HashSet<Long>) entity.getProperty("users");
-      HashSet<Long> subtasks = (HashSet<Long>) entity.getProperty("subtasks");
-      // Build TaskData object
-      TaskData task = new TaskData(taskID, projectID, name, description, status, users, subtasks);
+      if (count++ >= quantity) {
+        break;
+      }
```
[the remark this answers is not in the document]
Comment (Tzirliya) [reply src/main/java/com/rtb/projectmanagementtool/task/TaskController.java:39]: Do you know how to limit a query? I couldn't find anything that worked.

[the remark this answers is not in the document]
Comment (jcespejo) [reply src/main/java/com/rtb/projectmanagementtool/task/TaskController.java:39]: Try PreparedQuery FetchOptions:  https://cloud.google.com/appengine/docs/standard/java/javadoc/com/google/appengine/api/datastore/FetchOptions.html

[the remark this answers is not in the document]
Comment (jcespejo) [reply src/main/java/com/rtb/projectmanagementtool/task/TaskController.java:39]: Also if we have to do paging later on, limit+offset would work well for that.

**Pull request merged**